dynawo/sources/Common/DYNTrace.cpp

```diff
@@ -198,7 +198,12 @@ void Trace::addAppenders_(const std::vector<TraceAppender>& appenders) {
 
   {
     boost::lock_guard<boost::mutex> lock(mutex_);
-    sinks_.insert_or_assign(currentId, traceSink);
+
```

Comment:
WHy modifying this? It seems that insert_or_assign method is also defined in std

Reply:
insert_or_assign() is only available since C++17